### PR TITLE
fix(tools): add missing colors to the Color storybook page

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

https://github.com/freeCodeCamp/freeCodeCamp/pull/44970 exported color variables from the `colors.css` file in order to have them displayed on the Color storybook page. However, some new colors were missing from the export.

This PR is to complete the color list.

## To test

Run `npm run storybook` and check the Color story.

## Screenshot

| Before | After |
| --- | --- |
| Screen Shot 2022-02-04 at 22 32 20 | Screen Shot 2022-02-04 at 22 29 51 |

### CODE REVIEW

File: tools/ui-components/src/colors.css [hunk 1]
```diff
@@ -59,22 +59,34 @@
...
   --yellow20: var(--yellow20);
+  --yellow40: var(--yellow40);
+  --yellow45: var(--yellow45);
   --yellow50: var(--yellow50);
+  --yellow70: var(--yellow70);
   --yellow90: var(--yellow90);
 
+  --blue05: var(--blue05);
   --blue10: var(--blue10);
-  --blue20: var(--blue20);
```
Comment (huyenltnguyen) [inline tools/ui-components/src/colors.css:68]: `blue20` no longer exists.

https://github.com/freeCodeCamp/freeCodeCamp/blob/044f0f927832c3ac4d2fbdb54a3c188267620062/tools/ui-components/src/colors.css#L24-L29

File: tools/ui-components/src/colors.css [hunk 2]
```diff
@@ -59,22 +59,34 @@
...
+  --green05: var(--green05);
   --green10: var(--green10);
+  --green40: var(--green40);
+  --green70: var(--green70);
   --green90: var(--green90);
 
+  --red05: var(--red05);
   --red10: var(--red10);
+  --red15: var(--red15);
   --red20: var(--red20);
```
Comment (nhcarrigan) [inline tools/ui-components/src/colors.css:87]:
```suggestion
```

Looks like red20 also doesn't exist anymore?

File: tools/ui-components/src/colors.css [hunk 3]
```diff
@@ -59,22 +59,34 @@
   --purple50: var(--purple50);
   --purple90: var(--purple90);
 
+  --yellow05: var(--yellow05);
   --yellow10: var(--yellow10);
   --yellow20: var(--yellow20);
```
Comment (nhcarrigan) [inline tools/ui-components/src/colors.css:64]:
```suggestion
```

Also yellow20 it seems.

**Comment by huyenltnguyen** [comment]
Ugh. I swear I reviewed the list twice before opening the PR.

Great catch @nhcarrigan!

**Comment by nhcarrigan** [comment]
The screenshot you added definitely helped!

**Review by Sembauke — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Let's go ahead and get this in.

**Pull request merged**